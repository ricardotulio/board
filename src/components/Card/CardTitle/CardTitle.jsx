import React from 'react'

const CardTitle = ({ title }) => (
  <h3 className="card-title">
    {title}
  </h3>
)

export default CardTitle
