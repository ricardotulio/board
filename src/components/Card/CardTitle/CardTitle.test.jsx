import { render } from '@testing-library/react'
import '@testing-library/jest-dom'
import CardTitle from './CardTitle'

it('can render a title', () => {
  const title = 'card title'
  const { container, getByText } = render(<CardTitle title={title} />)
  expect(getByText('card title')).toBeInTheDocument()
  expect(container.firstChild).toMatchSnapshot()
})

