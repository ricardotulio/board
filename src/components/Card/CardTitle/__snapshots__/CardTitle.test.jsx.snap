// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`can render a title 1`] = `
<h3
  class="card-title"
>
  card title
</h3>
`;
